label_studio/tasks/api.py

```diff
@@ -182,6 +182,11 @@ def get(self, request, pk):
         return Response(data)
 
     def get_queryset(self):
+        task_id = self.request.parser_context['kwargs'].get('pk')
+        task = generics.get_object_or_404(
+            Task.objects.filter(id=task_id),
```

Comment:
why do we need this queryset here? why not `generics.get_object_or_404(Task, pk=task_id)` ?